src/stylesheets/components/_buttons.scss

```diff
@@ -79,14 +89,7 @@ a.btn--transparent--inverse {
 [class*="status--"] a.btn--small,
 [class*="theme--"] a.btn--small,
 .btn--small {
-  padding: 1px ($base-spacing-unit + 1) 0;
-}
-
-/*--- Button Component Rules ---*/
-
-.status--banner [class*="btn--"] {
-  @include micro;
-  text-decoration: none;
+  padding: 1px ($base-spacing-unit + 1);
```

Comment:
What's the `+ 1` about? Seems like a magic number - some context would be great :-)

Reply:
It just looks nicer.